examples/player/simid_player.js

```diff
@@ -271,9 +289,14 @@ class SimidPlayer {
     // Once the ad is successfully initialized it can start.
     // If the ad is not visible it must be made visible here.
     this.showSimidIFrame_();
-    this.showAdPlayer_();
-    this.adVideoElement_.src = document.getElementById('video_url').value;
-    this.adVideoElement_.play();
+
+    // only linear ones play ad videos
```

Comment:
make this descriptive for developers.
Suggestion:
// A linear ad will pause the content video and show the ad video. A nonlinear ad will overlay the content video.

A possibly more readable way you could code this would be:

if (this.isLinearAd_) {
  playLinearVideoAd_();
}
....

/**
 * Pauses the content video and starts playback of the ad video.
 */
 playLinearVideoAd_() {
      this.contentVideoElement_.pause();
      this.showAdPlayer_();
      this.adVideoElement_.src = document.getElementById('video_url').value;
      this.adVideoElement_.play();
}